### PR TITLE
new layout

### LINKED ISSUE #38: New app layout with footer
New design app layout:

image

Logos for footer (use svgs): https://drive.google.com/drive/folders/1nQ81oZsYM3_Hw0Pf4B4_Aycz7AJNUVeR

### PR DESCRIPTION
**Pull request by dlinah**
# Pull Request

## Description
implementing new layout
Fixes #38

## How Has This Been Tested?

visually

- [x] Yes

Screenshot 2022-05-16 at 15 41 00

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: apps/nowcasting-app/components/layout.tsx [hunk 1]
```diff
@@ -14,9 +14,10 @@ const Layout = ({ children }: ILayout) => {
         <title>Nowcasting App</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Navbar />
-      <main>{children}</main>
-      <Footer />
+      <main className="absolute inset-0 flex flex-col	">
```
Comment (Alaatohamy) [inline apps/nowcasting-app/components/layout.tsx:17]: we need linting plugin, remove white spaces

File: apps/nowcasting-app/components/side-layout.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const SideLayout: React.FC<SideLayoutProps> = ({ children, className }) => {
+  return <div className={"border-t border-black h-full text-white justify-between flex flex-col bg-mapbox-black-500 " + (className || "")}>
+    <div className="h-10 mb-auta">
+      {children}
+    </div>
+    <Footer />
+  </div>;
+};
+
+export default SideLayout;
```
Comment (Alaatohamy) [inline apps/nowcasting-app/components/side-layout.tsx:17]: new line

File: apps/nowcasting-app/components/layout.tsx (hunk 1 above)
Comment (dlinah) [reply apps/nowcasting-app/components/layout.tsx:17]: ofcourse 👌  ,can you add a story to config eslint or prettier

File: apps/nowcasting-app/components/side-layout.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Footer from './footer';
+
+type SideLayoutProps = {
+  className: string;
```
Comment (Alaatohamy) [inline apps/nowcasting-app/components/side-layout.tsx]: so change it in the types to optional, please

**Review by dctanner — APPROVED** [review]
Looks good

**Pull request merged**